runtime/vm/classloadersearch.c

```diff
@@ -228,14 +256,12 @@ addZipToLoader(J9JavaVM * vm, const char * filename, J9ClassLoader * classLoader
 			(*env)->CallVoidMethod(env, classLoaderRef, mid, filenameString);
 			if ((*env)->ExceptionCheck(env)) {
 				rc = CLS_ERROR_OUT_OF_MEMORY;
-			} else if (J2SE_SHAPE(vm) >= J2SE_SHAPE_B165) {
-				rc = loadModule(vm, INSTRUMENT_MODULE, NULL);
 			}
-	cleanup:
+cleanup:
 			(*env)->ExceptionClear(env);
-			(*env)->DeleteLocalRef(env, classLoaderRef);
-			(*env)->DeleteLocalRef(env, filenameString);
 			(*env)->DeleteLocalRef(env, classLoaderClass);
+			(*env)->DeleteLocalRef(env, filenameString);
+			(*env)->DeleteLocalRef(env, classLoaderRef);
```

Comment:
For my own education: why were these calls to `DeleteLocalRef` reordered?

Reply:
I reordered them to be in reverse order of creation, in case that were to ever matter for future optimization of the local ref space (I don't think it does at the moment).